src/lib.rs

```diff
@@ -129,15 +145,16 @@ impl Build {
     }
```

Comment:
We need an info log that tells the user the build has been completed successfully. We don't need to display anything if it fails because the errors from the build will make this clear enough.